doc/en/components/grids/data-grid.md

```diff
@@ -788,7 +719,7 @@ public smallViewTemplate = (ctx: IgcCellTemplateContext) => {
 ```
 
 ```tsx
-<IgrGrid>
+<IgrGrid ref={grid}>
```

Comment:
The `normalViewTemplate` and `smallViewTemplate` snippets have `class` instead of `className` set. Also, they are using `ctx.dataContext.cell.value` which is no longer the case and should be `ctx.cell.value`